test/e2e/common/setup-kubestellar.sh

```diff
@@ -127,6 +149,21 @@ else
 popd
 
 : Waiting for OCM hub to be ready...
+
+kubectl wait controlplane.tenancy.kflex.kubestellar.org/its1 --for 'jsonpath={.status.postCreateHooks.its-with-clusteradm}=true' --timeout 400s
```

Comment:
As you can see from the _conditional_ use of the other copy of this wait in this script, this is not always the right wait to do.